modules/site-icon/site-icon-functions.php

```diff
@@ -26,7 +26,17 @@ function jetpack_get_site_icon( $blog_id = null, $size = '512', $default = '', $
 	$alt = ( $alt ? esc_attr( $alt ) : __( 'Site Icon', 'jetpack' ) );
 	$src = esc_url( jetpack_site_icon_url( $blog_id, $size, $default ) );
 	$avatar = "<img alt='{$alt}' src='{$src}' class='$class' height='{$size}' width='{$size}' />";
-
+	/**
+	 * Filters the display options for the Site Icon.
+	 *
+	 * @since 3.2.0
+	 *
+	 * @param string $avatar The Site Icon in an html image tag.
+	 * @param int $blog_id The Blog ID.
```

Comment:
Can you add a couple of spaces between `int` and `$blog_id` so that the variables all align?

`$blog_id` is this the blog_id on the local site or the wp.com blog_id?